src/test/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileProcessorImplTest.java

```diff
@@ -61,65 +67,113 @@ void setUp() {
 
     @Test
     void processAudioForDownloadShouldReturnOneSessionWithOneAudioWhenProvidedWithOneAudio()
-        throws ExecutionException, InterruptedException, IOException {
-
-        AudioFileInfo trimmedAudioFileInfo = new AudioFileInfo();
-        when(audioOperationService.trim(any(), any(), any(), any()))
-            .thenReturn(trimmedAudioFileInfo);
+            throws ExecutionException, InterruptedException, IOException {
 
+        // Given
         var mediaEntity = createMediaEntity(
-            TIME_12_00,
-            TIME_12_10,
-            1
+                TIME_12_00,
+                TIME_12_10,
+                1,
+                1
         );
         var mediaEntityToDownloadLocation = Map.of(mediaEntity, SOME_DOWNLOAD_PATH);
 
+        AudioFileInfo originalAudioFileInfo = AudioFileInfo.builder()
+                .startTime(TIME_12_00.toInstant())
+                .endTime(TIME_12_10.toInstant())
+                .channel(1)
+                .mediaFile("0001.a00")
+                .path(SOME_DOWNLOAD_PATH)
+                .isTrimmed(false)
+                .build();
+
+        AudioFileInfo trimmedAudioFileInfo = AudioFileInfo.builder()
+                .startTime(TIME_12_02.toInstant())
+                .endTime(TIME_12_09.toInstant())
+                .channel(1)
+                .mediaFile("0001.a00")
+                .path(SOME_DOWNLOAD_PATH)
+                .isTrimmed(true)
+                .build();
+
+        when(audioOperationService.trim(
+                "",
+                originalAudioFileInfo,
+                Duration.ofMinutes(2),
+                Duration.ofMinutes(9)))
+                .thenReturn(trimmedAudioFileInfo);
+
+        // When
         List<List<AudioFileInfo>> sessions = outboundFileProcessor.processAudioForDownload(
-            mediaEntityToDownloadLocation,
-            TIME_12_00,
-            TIME_12_10
+                mediaEntityToDownloadLocation,
+                TIME_12_02,
```

Comment:
just wondering why checkstyle doesn't pick up on all these indentation variations?

Reply:
No idea - Pass.  Maybe it's not a standard checkstyle concern.
We have a simple https://github.com/hmcts/darts-api/blob/master/.editorconfig
But I believe we are supposed to be using `GoogleStyle` Code Style - hopefully mine hasn't reverted to something else as I did upgrade my IntelliJ version recently